content/tutorials/integration-tutorials/keras.md

```diff
@@ -36,17 +38,20 @@ import wandb
 from wandb.integration.keras import WandbMetricsLogger
 ```
 
-If this is your first time using W&B or you are not logged in, the link that appears after running `wandb.login()` will take you to sign-up/login page. Signing up for a [free account](https://wandb.ai/signup) is as easy as a few clicks.
-
+If this is your first time using W&B or you are not logged in, the link that
+appears after running `wandb.login()` will take you to sign-up/login page.
+Signing up for a [free account](https://wandb.ai/signup) is as easy as a few
+clicks.
 
 ```python
 wandb.login()
 ```
 
 ## Hyperparameters
 
-Use of proper config system is a recommended best practice for reproducible machine learning. We can track the hyperparameters for every experiment using W&B. In this colab we will be using simple Python `dict` as our config system.
-
+Use of proper config system is a recommended best practice for reproducible
+machine learning. We can track the hyperparameters for every experiment using
+W&B. In this colab we will be using simple Python `dict` as our config system.
```

Comment:
```suggestion
To reproduce a machine learning experiment, it's important to save the hyperparameters that the experiment uses. This example shows how to track an experiment's hyperparameters in a Python `dict`.
```
This sentence is very hard for me to understand (not due to your changes). Here is my attempt to help, which may in fact be incorrect.